gatsby-node.ts

```diff
@@ -10,6 +10,13 @@ import { loadDocuments, loadSchema } from 'graphql-toolkit';
 import { GatsbyNode } from 'gatsby';
 // @ts-ignore
 import { graphql, introspectionQuery } from 'gatsby/graphql';
+// @ts-ignore
+import getGatsbyDependents from "gatsby/dist/utils/gatsby-dependents.js"
+// @ts-ignore
+import report from "gatsby-cli/lib/reporter"
```

Comment:
If I understood this correctly, the reporter already included in [Node API helpers](https://www.gatsbyjs.org/docs/node-api-helpers/#reporter), so we don't need to import gatsby-cli for it.

Reply:
Indeed! I have rewritten the code to use the included reporter.